### DISCUSSION TITLE
"shadow" in source code textarea makes the source code hard to read

### DISCUSSION THREAD

**Discussion by voidzero** [community/community#101399] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Every source code looks like the following for me:

![image](https://github.com/community/community/assets/1332504/19c97731-2065-4041-ab4b-e57a56ca319f)

When I am in the light layout, the text looks all black.

when I go into my browser's dev tools and disable `* text-shadow { ... }`, in the following image at the bottom:

![image](https://github.com/community/community/assets/1332504/caa0311a-a284-4656-9709-504f33eeaece)

It fixes the problem:

![image](https://github.com/community/community/assets/1332504/b4607f61-fe87-414d-b4ab-5f5cfe042045)

But it's kind of a hassle to do this every time. Is this an issue on my end (I don't see how, because I don't have anything installed afaik that causes this) or is this something that others are experiencing too? What can I do to fix it, any suggestions?

**Comment by voidzero** [comment]
Any takers? Please and thank you 😬

**Comment by voidzero** [comment]
@queenofcorgis Do you have advice for me pretty please?

**Reply by queenofcorgis** [reply]
Hi there,

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our Code of Conduct for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

**Comment by voidzero** [comment, marked as answer]
Found the culprit: Font Rendering Enhancer. Disabling that extension solved the issue.

**Comment by queenofcorgis** [comment]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.
